helm_tests/webserver/test_webserver.py

```diff
@@ -1154,3 +1154,86 @@ def test_overridden_automount_service_account_token(self):
             show_only=["templates/webserver/webserver-serviceaccount.yaml"],
         )
         assert jmespath.search("automountServiceAccountToken", docs[0]) is False
+
+
+class TestWebserverHPAAutoScaler:
+    """Tests webserver HPA auto scaler."""
+
+    def test_should_add_component_specific_labels(self):
```

Comment:
Can you move this to `test_hpa_webserver`?

Reply:
Sure!